lib/rubocop/ast/node/block_node.rb

```diff
@@ -32,6 +35,16 @@ def arguments
         end
       end
 
+      # Returns a collection of all descendants of this node that are
+      # argument type nodes. See `ArgsNode#argument_list` for details.
+      #
+      # @return [Array<Node>]
+      def argument_list
+        return [] unless arguments?
```

Comment:
@marcandre is there an array other than this one that you want me to freeze? Not seeing anything else.

Reply:
Actually I think you meant `arguments_list` so I've frozen that as well now.